js/app.js

```diff
@@ -0,0 +1,81 @@
+//person4------------------------------------------------------
+const person4 = new XMLHttpRequest();
+function person4Req() {
+  let person4Obj = JSON.parse(this.responseText);
+  document.getElementById("person4Name").innerHTML =
+    "Person 4: " + person4Obj.name;
+}
+person4.addEventListener("load", person4Req);
+person4.open("GET", "https://swapi.co/api/people/4/");
+person4.send();
+
+const person4Homeworld = new XMLHttpRequest();
+function person4HomeReq() {
+  let homeworld = JSON.parse(this.responseText);
+  document.getElementById("person4HomeWorld").innerHTML =
+    "HomeWorld: " + homeworld.name;
+}
+person4Homeworld.addEventListener("load", person4HomeReq);
+person4Homeworld.open("GET", "https://swapi.co/api/planets/1/");
+person4Homeworld.send();
```

Comment:
You should be making this request from the event handler from getting person 4. The idea is that you normally wouldn't know what person 4's homeworld is until you got person 4's data.